screens/foodTypes.js

```diff
@@ -0,0 +1,130 @@
+import { useState } from "react";
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  FlatList,
+  StyleSheet,
+} from "react-native";
+import { useTailwind } from "tailwind-rn";
+
+const FoodTypes = ({ closeFoodTypes }) => {
```

Comment:
Also lets keep naming consistent instead of calling this component and file `foodTypes`. lets call both the component and file `Search` since this screen will be for searching, not for foodTypes.